contracts/components/node_runners/NodeRunnerRegistry.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-network/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.9;
+
+import "../BaseComponentUpgradeable.sol";
+import "./NodeRunnerRegistryCore.sol";
+import "./NodeRunnerRegistryManaged.sol";
+
+/**
+ * ERC721 Registry of Node Runners. Each node runner controls a number of Scanner Nodes, represented by their EOA address.
+ * NodeRunner must register themselves, then register node addresses to be controlled by their NodeRunner ID (incremental uint). Registered NodeRunners can also assign managers to manage the nodes.
+ * Each scanner has a single "chainId" and metadata (string that can point to a URL, IPFS…). Node runners and managers can update said metadata.
+ * Scanner Nodes can be enabled or disabled by:
+ * - the Scanner itself,
+ * - the NodeRunner
+ * - any of the scanner managers
+ * If the scannerId is staked under the minimum stake, it can’t be `enabled()` and `isEnabled()` will return false, regardless of the disabled flag.
```

Comment:
It's weird that "disabled" and "enabled" are not opposite concepts, as seen for example in the getters `isDisabled` and `isEnabled`.

I think we're conflating things. I would give the concepts different names and make a stricter distinction. I would propose to use _disabling_/_enabling_ to refer to the action that is done manually, and use _operational_ to refer to the property that depends on stake and stake requirements. This would imply renaming `isEnabled` to `isOperational`.

I would make these concepts completely separate, for example the event `ScannerEnabled` should not include whether the scanner is operational or not, as that's something that has to be analyzed separately anyway.

Reply:
I agree is very akward but chaning the names in the interface would break many off-chain things, and one of the design principles I got for this is to be as less disruptive with ABIs as possible to minimize the need to coordinate a simultaneous deployment of node software, CLIs, assigner, etc risking downtime.

Personally I thing we should take time to make the software more resilient to ABI changes adding a layer of indirection in off chain software, maybe something that get the ABIs from some kind of public decentralized repo (open to suggesion if something like that exists), but it's not a priority at the time.

So I think I can't touch `ScannerEnabled` event, but now that I think about it, since this is a new contract and `ScannerNodeRegistry` is going to be kept as compatibility layer, maybe this is the time to rename the methods.

Reply:
For the sake of better contracts, I'm going to introduce `isOperational`, I'll try to keep it compatible using  `ScannerNodeRegistry` and present it to the team, noting `ScannerEnabled` stays the same. (`isEnabled` is also in the AgentRegistry contracts, but that may be a change for "another day"). If it doesn't fly I'll rever the change.

12fbeba